test/test_traceback_leak.py

```
@@ -0,0 +1,182 @@
+the ``broker``.  This creates a circular reference chain::
+
+    broker.exceptions -> exception -> __traceback__ -> frame -> broker
+
+``SentryMonitoredBroker.add_exception()`` clears ``ex.__traceback__``
+after Sentry captures the exception and the formatted traceback string
+is stored, breaking the cycle without losing any debugging information.
+"""
+
+import gc
+import platform
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)